cpp/diffraction/StaticStructureFactorDebye.h

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2010-2020 The Regents of the University of Michigan
+// This file is from the freud project, released under the BSD 3-Clause License.
+
+#ifndef STATIC_STRUCTURE_FACTOR_DEBYE_H
+#define STATIC_STRUCTURE_FACTOR_DEBYE_H
+
+#include "Box.h"
+#include "Histogram.h"
+#include "NeighborQuery.h"
+
+/*! \file StaticStructureFactorDebye.h
+    \brief Routines for computing static structure factors.
+
+    Computes structure factors from the Fourier transform of a radial distribution function (RDF).
+    This method is not capable of resolving k-vectors smaller than the magnitude 4 * pi / L, where L is the
+    smallest side length of the system's periodic box.
+*/
+
+namespace freud { namespace diffraction {
+
+class StaticStructureFactorDebye
+{
+    using StaticStructureFactorDebyeHistogram = util::Histogram<float>;
+
+public:
+    //! Constructor
+    StaticStructureFactorDebye(unsigned int bins, float k_max, float k_min = 0);
+
+    //! Destructor
+    virtual ~StaticStructureFactorDebye() = default;
```

Comment:
Do we plan on having classes derived from this one? Right now there isn't a need to make this virtual.

Reply:
I don't think so? @bdice

Reply:
It is probably unnecessary to define the destructor. Try deleting it. I think this class can adhere to the "Rule of zero": https://en.cppreference.com/w/cpp/language/rule_of_three

Reply:
Deleted.